### PR TITLE
Add methods for getting cursor position from split editor

### PR DESCRIPTION
**Pull request by musienko-maxim**
### What does this PR do?
* Add methods for correct getting  cursor position from split editors
* Fix CheckRestoringSplitEditorTest selenium test
* Rename Editor class (Codenvy preffix is not actual)
### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/7705
@tolusha, @vparfonov, @dmytro-ndp

### CODE REVIEW

**Comment by musienko-maxim** [comment]
ci-test

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/Editor.java [hunk 1]
```diff
@@ -0,0 +1,1914 @@
...
+
+    /**
+     * set cursor in specified position for current visible editor
+     *
+     * @param positionLine
+     *         is the specified number line
+     * @param positionChar
+     *         is the specified number char
+     */
+    public void setCursorToDefinedLineAndCharForCurrentVisibleEditor(int positionLine, int positionChar) {
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/Editor.java]: Method name **setCursorToDefinedLineAndCharForCurrentVisibleEditor()** looks to long, IMHO.
Can we use here **goToPosition()**?

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/Editor.java [hunk 2]
```diff
@@ -0,0 +1,1914 @@
...
+
+    /**
+     * wait specified values for Line and Char cursor positions in the Codenvy editor
+     *
+     * @param linePosition
+     *         expected line position
+     * @param charPosition
+     *         expected char position
+     */
+    public void waitLineAndCharForCurrentVisibleEditor(final int linePosition, final int charPosition) {
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/Editor.java]: IMHO, it's better to rename the method to have a shorter name **waitCursorInPosition()** .

**Comment by dmytro-ndp** [comment]
@musienko-maxim: if **Editor** page object operates with the current active visible editor only, the words "_VisibleEditor_", "_CurrentVisibleEditor_", "_ActiveEditor_" look redundant in the methods' names and so could be omitted with saving the meaning of methods, IMHO.
If it linked to split editors, we should select the only one word between the `{Active, Visible, Current}`. As for me, **Active** is the most suitable for this case.

**Comment by tolusha** [comment]
It could be nice of to split commit into several ones not to spend time browsing through refactoring stuff.

**Pull request closed without merging**